### ISSUE TITLE
Whitelist server compatability.

### ISSUE THREAD

**Issue by ConfuseN** [DiscordSRV/RequireLinked4Gameplay#2]
I would like to see this plugin work with whitelists more easily, or have some sort of whitelist mode. 
Currently when you install the plugin and try to connect to a server (and you are unwhitelisted) you get the code to link your account, rather then the "you are not whitelisted message" This creates confusion for people trying to register, especially if your server does whitelist applications through discord (EG: a link to the app) 

For me I give my whitelisted players the member role whenever they link their account, however now unwhitelisted members can get the member role.

For now i've been using an older version of the plugin that prevents movement, but would really love to see some kind of support for whitelisted servers!

**Comment by Scarsz** [comment]
I’m confused with what specifically you’re asking for. Do you want it to work with the vanilla server’s whitelist? Other whitelist plugins?

**Comment by burturt** [comment]
While I know it's been a while, I think he wants an option to have this plugin kick players and give them the link code AFTER minecraft's whitelist detects and possibly kicks players for not being whitelisted.

**Comment by Vankka** [comment]
probably more like a bypass for the whole system, eg. whitelisted users don't need to link at all

**Comment by pkrok01** [comment]
tldr; run a whitelist check before running the linked check
